internal/page/certificateprovider/unable_to_confirm_identity.go

```diff
@@ -0,0 +1,47 @@
+package certificateprovider
+
+import (
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/lpastore"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type unableToConfirmIdentityData struct {
+	App    page.AppData
+	Donor  lpastore.Donor
+	Errors validation.List
+}
+
+func UnableToConfirmIdentity(tmpl template.Template, certificateProviderStore CertificateProviderStore, lpaStoreResolvingService LpaStoreResolvingService) page.Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request) error {
+		if r.Method == http.MethodPost {
+			certificateProvider, err := certificateProviderStore.Get(r.Context())
+			if err != nil {
+				return err
+			}
+
+			certificateProvider.Tasks.ConfirmYourIdentity = actor.TaskCompleted
```

Comment:
We can see how Beth feels about this in review - setting to there's a problem or something similar didn;t feel right as the CP can still sign.

Reply:
I agree with this, you've completed the work for the task even if you haven't confirmed your identity